.github/workflows/e2e-azure-management-and-workload-cluster.yaml

```diff
@@ -0,0 +1,49 @@
+name: E2E Test - Azure Management and Workload Cluster
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.go"
+      - "cli/cmd/plugin/**/go.mod"
+      - "**.sh"
+      - ".github/workflows/e2e-azure-management-and-workload-cluster.yaml"
+    tags-ignore:
+      - "**"
+
+jobs:
+  e2e-azure-management-and-workload-test:
+    name: E2E Azure Management and Workload Cluster Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
+        id: go
+
+      - name: Cleaning up Github Actions runner
+        run: |
+          df -h
+          sudo rm -rf /usr/local/lib/android
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf "/usr/local/share/boost"
+          df -h
```

Comment:
What is the reason for this section? I don't think any of those paths exist for us.

And GitHub is mis-capitalized. ;)

Reply:
This section is closely tied to the VM / test environment where the test runs. Previously we used to run on GitHub hosted runners which had these paths and we had space issues, and we followed this - https://github.com/actions/virtual-environments/issues/2606#issuecomment-772683150 . Now that we use self hosted runners, we can get rid of these, yeah. It's just dead code now